src/utils/auth.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+const jwt = require('jsonwebtoken');
+
+// TODO: Test out this middle where once login endpoint is done and returns a JSON web token
+const authenticateJWT = (req: any, res: any, next: () => void) => {
+    const authHeader = req.headers.authorization;
+
+    if (authHeader) {
+        const token = authHeader.split(' ')[1];
+        try {
+            jwt.verify(token, process.env.ACCESS_TOKEN_SECRET)
+            .then( () => {
+                next();
+            })
+
+        } catch (e: any) {
```

Comment:
```suggestion
        } catch (e: Error) {
```

Reply:
It wouldn't let me type `e` with `Error`. I think something is up with typing req, res, and next. When I tried to type it with express, TS did not like me trying to assign `req.user`.

Reply:
I just took the any and Error types away and TS was fine with it. When I type it to Error is gives an error message: "Catch clause variable type annotation must be 'any' or 'unknown' if specified."